nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -52,7 +52,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "{{ cookiecutter.app_name }}": {
-            "nautobot_ver": "2.3.1",
+            "nautobot_ver": "{{ cookiecutter.min_nautobot_version }}",
```

Comment:
This was hardcoded intentionally due to the fact that bumping min_nautobot_version only affects new repositories. I believe you should set this to the version you used above for `min_nautobot_version`.